Formula/vrpn.rb

```diff
@@ -22,18 +22,10 @@ class Vrpn < Formula
   depends_on "doxygen" => :build if build.with? "docs"
   depends_on "libusb" # for HID support
 
-  # Upstream commit fixes Xcode 9 build failure, remove in next version
-  # https://github.com/vrpn/vrpn/issues/187
-  patch do
-    url "https://github.com/vrpn/vrpn/commit/855af40d.patch?full_index=1"
-    sha256 "601335e211cee722814650a6a31cfc6a4278ae52d5e9527a2799df3dd47758b9"
-  end
-
   def install
-    ENV.libstdcxx
```

Comment:
It was originally added to fix build on Mavericks in 150a39ec33e3c741e39b67d5843c4d7e1c8979a3. It will now, however, cause build failures due to `std::swap not found` as Apple is killing `libstdc++`.

Reply:
Does it need to remain conditionally so we don't break 10.9, or?

Reply:
Changed in https://github.com/Homebrew/homebrew-core/pull/31183/commits/8a50560e165e328f7b12b974d0ada333c930a335#diff-ce622e10d121180b84ac0b43005bb311R26.